### PR TITLE
Add DataFrames.jl to README.md

### LINKED ISSUE #87: Add link to DataFrames.jl to README and docs
New Julia users repeatedly ask on forums for the equivalent of Pandas in Julia. Given that DataFrames.jl is the de-facto standard in native Julia, it would be awesome if the README of Pandas.jl could cite the package at the very top, possibly highlight the link.

Example of question: https://discourse.julialang.org/t/pandas-equivalent-library/26501

### PR DESCRIPTION
**Pull request by juliohm**
Fix #87

### CODE REVIEW

**Comment by juliohm** [comment]
cc: @malmaud

File: README.md [hunk 1]
```diff
@@ -5,7 +5,9 @@ Pandas.jl
 
 ![CI status](https://github.com/JuliaPy/Pandas.jl/actions/workflows/ci.yml/badge.svg)
 
-This package provides a Julia interface to the excellent [Pandas](http://pandas.pydata.org/pandas-docs/stable/) package. It sticks closely to the Pandas API. One exception is that integer-based indexing is automatically converted from Python's 0-based indexing to Julia's 1-based indexing.
+This package provides a Julia interface to the excellent [Pandas](http://pandas.pydata.org/pandas-docs/stable/) package from Python. It sticks closely to the Pandas API. One exception is that integer-based indexing is automatically converted from Python's 0-based indexing to Julia's 1-based indexing.
+
+For a pure Julia alternative, please check the excellent [DataFrames.jl](https://github.com/JuliaData/DataFrames.jl) package.
```
Comment (malmaud) [inline README.md:10]: nit, 'check' should be 'check out'.

**Comment by juliohm** [comment]
ping @malmaud

**Comment by malmaud** [comment]
thanks!

**Pull request merged**